client/accounts_test.go

```diff
@@ -0,0 +1,122 @@
+package client
```

Comment:
What do you think about using table-driven test as in the other test functions? 

https://github.com/golang/go/wiki/TableDrivenTests

Reply:
Yeah. But the use case is when there are many different test data sets and you want to avoid DRY. For this 3 cases it could be done, but for me it is clearer as it is.

Reply:
Since the other tests have used table driven approach, perhaps it would be best to do so for consistency sake if nothing else?

Reply:
Will try, but this is not a pattern that has to be applied always.

Reply:
This is specially true when assertion statements differ. Then TableDrivenTest pattern is not the most appropriate. 

For `account_test` there are three tests. Two of them are similar and can be implemented following TableDriveTest, but one of them (happy path test)  has different assertions. It could be implemented with specific assertion function for each test case... but for the implementation is less clear.